platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationConfigInstaller.java

```diff
@@ -76,59 +76,39 @@ public ApplicationConfigInstaller(
   }
 
   @Override
+  @SuppressWarnings("squid:S2093" /* Intentionally closing InputStream in finally clause */)
```

Comment:
Why are we closing this in a finally? Looks like we could just wrap only the properties section in a try with resource and then just return on error and avoid this suppression.

Reply:
After reviewing this with @paouelle we decided the finally block is the way to go.

1. It is an input stream so we don't care about if it cannot close or not
2. If everything works well in the `try with resources` and the `InputStream` has an error closing, an `IOException` is thrown and a misleading logger is logged: `LOGGER.warn("Could not load file as property list.", ioe);`

Reply:
OK. I would still recommend only wrapping the creation of the properties in the try/catch/finally

Reply:
Better yet replace all of the property loading stuff with `PropertiesLoader.getInstance().loadProperties(filename)`

Reply:
Doesn't that one do a lot more then just loading properties? It also performs system variable substitution

Reply:
I don't think that should be a problem with this properties file. It does provide a bit more functionality though. All in all I think it is a positive change.

Reply:
Then I am good with it too.